metadata-ingestion/src/datahub/ingestion/sink/datahub_rest.py

```diff
@@ -64,8 +69,15 @@ class RestSinkMode(ConfigEnum):
 )
 
 
+_DEFAULT_REST_SINK_ENDPOINT = pydantic.parse_obj_as(
+    RestSinkEndpoint,
+    os.getenv("DATAHUB_REST_SINK_DEFAULT_ENDPOINT", RestSinkEndpoint.OPENAPI),
```

Comment:
what's the first GMS version that supported openapi v3?

Reply:
Of note is that timeseries aspects for ingestion didn't work until 0.15. I'm fine with switching the default back to rest.li for now. The other aspects have been supported since ~0.13

Reply:
I think that's ok - especially as this will be part of 1.0

Reply:
Ok, I believe there is an issue with ASYNC timeseries aspects for openapi. That needs to be before it can be default. I've reverted to RESTLI.